### PR TITLE
1934 glassfish is installed and run as root

### PR DESCRIPTION
**Pull request by donsizemore**
## New Contributors

Welcome! New contributors should at least glance at [CONTRIBUTING.md](/CONTRIBUTING.md), especially the section on pull requests where we encourage you to reach out to other developers before you start coding. Also, please note that we measure code coverage and prefer you write unit tests. Pull requests can still be reviewed without tests or completion of the checklist outlined below. Thanks!

## Related Issues

- connects to #1934 : Glassfish is installed and run as root

## Pull Request Checklist

- [ ] Unit [tests][] completed
- [ ] Integration [tests][]: None
- [ ] Deployment requirements, [SQL updates][], [Solr updates][], etc.: None
- [ ] [Documentation][docs] completed
- [ ] Merged latest from "develop" [branch][] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: https://github.com/IQSS/dataverse/tree/develop/scripts/database/upgrades
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/4.6.0/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

File: doc/sphinx-guides/source/installation/prerequisites.rst [hunk 1]
```diff
@@ -43,40 +43,53 @@ Glassfish Version 4.1 is required. There are known issues with Glassfish 4.1.1 a
 Installing Glassfish
 ====================
 
-**Important**: once Glassfish is installed, a new version of the Weld library (v2.2.10.SP1) must be downloaded and installed. This fixes a serious issue in the library supplied with Glassfish 4.1 ( see https://github.com/IQSS/dataverse/issues/647 for details). Please note that if you plan to front Glassfish with Apache you must also patch Grizzly as explained in the :doc:`shibboleth` section.
```
Comment (oscardssmith) [inline doc/sphinx-guides/source/installation/prerequisites.rst]: Is this no longer true?

Comment (oscardssmith) [inline doc/sphinx-guides/source/installation/prerequisites.rst]: Never mind. It's just lower now

**Comment by kcondon** [comment]
@donsizemore Can you refresh this from develop? It's still using v4.7 as a version. Plan to merge this today. Leonid had planned to leave a couple comments on your doc changes but I think otherwise fine. Thanks!

**Comment by donsizemore** [comment]
@landreev I had intended for vagrant to install as GLASSFISH_USER but ran into permissions problems within Vagrant. I should've pulled that variable and missed it. How do things look now?

**Comment by landreev** [comment]
@donsizemore I left 2 comments in the issue (https://github.com/IQSS/dataverse/issues/1934); just realized that you no longer own it - so wasn't sure if you saw the updates.

**Pull request merged**